lib/biokbase/log.py

```diff
@@ -0,0 +1,374 @@
+"""
```

Comment:
Where did this library come from?  Is it something we can install from pip instead of including in the repo?

Reply:
I don't know what that came from, but the code had a dependency on it. Its under biokbase, so my guess is that it is KBase code.

Reply:
It's OG SDK python server code. It originally lived in `kbapi-common` but it's been updated a couple times and sits in a couple repos. I think it lives in the sdkbase2 docker image repo now.

Reply:
Yeah... I think this was pulled from the python sdk base.